django_x509/base/admin.py

```diff
@@ -59,11 +60,25 @@ class CaAdmin(BaseAdmin):
     list_filter = ['key_length', 'digest', 'created']
 
 
+class BaseX509Form(forms.ModelForm):
+        OPERATION_CHOICES = (
+            ('', ''),
+            ('import', 'import'),
+            ('new', 'new')
+        )
+
```

Comment:
Unnecessary line. Delete the line

Reply:
This line is needed as flake8 complains otherwise